src/ARMarker.js

```diff
@@ -24,7 +26,9 @@ class ARMarker extends HTMLElement {
                 src: arContent.getAttribute('src'),
                 scale: this.stringToVec3(arContent.getAttribute('scale')),
                 position: this.stringToVec3(arContent.getAttribute('position')),
-                rotation: this.stringToVec3(arContent.getAttribute('rotation'))
+                rotation: this.stringToVec3(arContent.getAttribute('rotation')),
+                loop: arContent.getAttribute('loop'),
```

Comment:
Here you can use something inline if:
```suggestion
                loop: arContent.getAttribute('loop'),
                loop: (arContent.getAttribute('loop') == "true"  ? true : false),
```